src/login.py

```diff
@@ -22,17 +21,57 @@ def __init__(self, browser: Browser, args: argparse.Namespace):
         self.utils = browser.utils
         self.args = args
 
-    def login(self) -> None:
-        if self.utils.isLoggedIn():
-            logging.info("[LOGIN] Already logged-in")
-        else:
-            logging.info("[LOGIN] Logging-in...")
-            self.executeLogin()
-            logging.info("[LOGIN] Logged-in successfully !")
-
-        assert self.utils.isLoggedIn()
+    def check_locked_user(self):
+        try:
+            element = self.webdriver.find_element(By.XPATH, "//div[@id='serviceAbuseLandingTitle']")
+            self.locked(element)
+        except NoSuchElementException:
+            return
+
+    def check_banned_user(self):
+        try:
+            element = self.webdriver.find_element(By.XPATH, '//*[@id="fraudErrorBody"]')
```

Comment:
Just curious, what is the source for these ids? Did you have this information previously? Or did you sacrifice an account for this haha.